test/unit/BPool.t.sol

```diff
@@ -57,6 +57,10 @@ abstract contract BasePoolTest is Test, BConst, Utils {
     bPool.set__records(_token, _record);
   }
 
+  function _setBalance(address _user, uint256 _balance) internal {
+    _writeUintAtAddressMapping(address(bPool), 0, _user, _balance);
+  }
```

Comment:
please move next to totalSupply, and add the comment on why we're using this method and not smock one

Reply:
i wonder, should we use `vm.deal(address token, address to, uint256 amount)`? i wonder if that increases total supply, could you confirm?